src/main/java/com/ProjectEureka/backend/models/Answer.java

```diff
@@ -0,0 +1,34 @@
+package com.ProjectEureka.backend.models;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Document(collection = "Answers")
+public class Answer {
+    @Id
+    private String id;
+    @Field("media_urls")
+    private List<String> mediaUrls;
+    @Field("answer_date")
+    private String answerDate;
+    private String description;
+    @Field("question_id")
+    private Long questionId;
+    @Field("user_id")
+    private Long userId;
```

Comment:
Please make a new line in between each variable

Reply:
Fixed spacing in new push